assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -318,60 +246,21 @@ describe( 'AudienceTilesWidget', () => {
 			isAudienceSegmentationWidgetHidden: false,
 		} );
 
-		const { container, waitForRegistry } = render(
-			<WidgetWithComponentProps />,
-			{
-				registry,
-			}
-		);
-
-		await waitForRegistry();
-
-		expect( container ).toBeEmptyDOMElement();
-	} );
-
-	it( 'should not render when configuredAudiences is null (not set)', async () => {
-		registry
-			.dispatch( MODULES_ANALYTICS_4 )
-			.setAvailableAudiences( availableAudiences );
+		freezeFetch( syncAvailableAudiencesEndpoint );
 
-		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
-			configuredAudiences: null,
-			isAudienceSegmentationWidgetHidden: false,
-		} );
-
-		const { container, waitForRegistry } = render(
-			<WidgetWithComponentProps />,
-			{
-				registry,
-			}
-		);
-
-		await waitForRegistry();
-
-		expect( container ).toBeEmptyDOMElement();
-	} );
+		await act( async () => {
+			// Render the component and wait for the registry updates
+			const { container, waitForRegistry } = render(
+				<WidgetWithComponentProps />,
+				{
+					registry,
+				}
+			);
 
-	it( 'should not render when there is no matching audience', async () => {
```

Comment:
It would be good to keep this test and repurpose it to verify that the "no audiences" banner is rendered. I'll push the change myself. to save the round trip.